### PR TITLE
CLIMATE-773 - Add Continuous Integration (Travis-CI)

### PR DESCRIPTION
**Pull request by jarifibrahim**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+language: python
```
Comment (lewismc) [inline .travis.yml]: Apache License header file needs to be added here.

**Comment by lewismc** [comment]
Do you have evidence of this running and it being stable @jarifibrahim ?

**Comment by jarifibrahim** [comment]
@lewismc Yes. The Travis build fails because of the bad tests but Travis works as expected. I've created a Travis-test branch which works fine after fixing the tests.

**Comment by lewismc** [comment]
Thanks

**Comment by lewismc** [comment]
Thanks @jarifibrahim

**Pull request closed without merging**